Makefile

```diff
@@ -69,11 +69,10 @@ docs: crate stdlibdocs ## Assemble all the /docs for the website locally.
 	cp -rv README.md docs/index.md # Special copy for the main README
 	cp -rv api/README.md docs/stdlib/index.md # Special copy for the main README
 
- ifeq (, $(shell which protoc-gen-doc))
- $(error "No protoc-gen-doc, install from https://github.com/pseudomuto/protoc-gen-doc")
- endif
-
 stdlibdocs: ## Generate the docs for the stdlib from the .proto files
+	ifeq (, $(wildcard /usr/local/bin/protoc-gen-doc))
```

Comment:
Why is this `$(wildcard` and not `$(shell`?

Reply:
it does.. and it saves a shell out.